controllers/shipwrightbuild_controller.go

```diff
@@ -219,10 +218,9 @@ func (r *ShipwrightBuildReconciler) setupManifestival(managerLogger logr.Logger)
 	if err != nil {
 		return err
 	}
-	buildManifest := filepath.Join(dataPath, "release.yaml")
 
 	r.Manifest, err = manifestival.NewManifest(
-		buildManifest,
+		dataPath,
```

Comment:
I think the trigger is the option component, so it would be great if do not deploy trigger by default.

Reply:
It is now optional, regarding if we set an env var to install the triggers or not